package.json

```diff
@@ -18,5 +18,8 @@
      "devDependencies": {
           "concurrently": "^4.0.1",
           "recursive-install": "^1.4.0"
+     },
+     "dependencies": {
+          "react-chat-ui": "^0.3.2"
```

Comment:
The hierarchy of this repo is kind of difficult. So:

package.json -- top level development dependencies
   src/client/package.json -- dependencies for frontend (probably where this belongs)
   src/server/package.json -- dependencies for backend/Express server

Reply:
fixed!

Reply:
You may need to run `npm unstainll react-chat-ui` at the top level directory to remove this change (or just deletethese lines from the JSON)